guardian-ui/src/App.tsx

```diff
@@ -1,22 +1,145 @@
+import React, { useContext } from 'react';
 import {
 	ChakraProvider,
 	Box,
-	Text,
 	VStack,
-	Grid,
 	theme,
+	Button,
+	Heading,
 } from '@chakra-ui/react';
 import { ColorModeSwitcher } from './ColorModeSwitcher';
+import {
+	ACTION_TYPE,
+	GuardianContext,
+	GuardianProvider,
+	UserType,
+} from './context/context';
+
+export const App: React.FC = () => {
+	return (
+		<ChakraProvider theme={theme}>
+			<GuardianProvider>
+				<Box textAlign='center' fontSize='xl'>
+					<ColorModeSwitcher justifySelf='flex-end' />
+					<VStack spacing={8}>
+						<GuardianFlow />
+					</VStack>
+				</Box>
+			</GuardianProvider>
+		</ChakraProvider>
+	);
+};
```

Comment:
If we're going to use tabs, can we figure out how to configure GitHub to display them better? This is kinda hard to read with 8-space indentation.

Reply:
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/managing-your-tab-size-rendering-preference

You can update it in your personal settings.